swap_meet/vendor.py

```diff
@@ -1,2 +1,66 @@
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        if not inventory:
+            inventory = [] 
+        self.inventory = inventory
+
+    def add(self, add_item):
+        self.add_item = add_item
```

Comment:
This line will create an attribute `add_item` in this class.  This attribute is not used anywhere else and is not needed.